examples/SuperResolution/SuperResolution.cpp

```diff
@@ -0,0 +1,122 @@
+    const wnn::Operand convBias = BuildConstantFromNpy(builder, biasPath);
+
+    const wnn::Conv2dOptions* conv2dOptions = options != nullptr ? options->AsPtr() : nullptr;
+    const wnn::Operand conv2d = builder.Conv2d(input, convWeights, conv2dOptions);
+
+    if (!mFused) {
+        if (relu) {
+            return builder.Relu(conv2d);
+        }
+        return conv2d;
+    }
+
+    // Fused
+    utils::Conv2dOptions fusedOptions;
+    if (options != nullptr) {
+        fusedOptions = *options;
+    }
+    fusedOptions.bias = convBias;
+
+    if (relu) {
+        fusedOptions.activation = builder.ReluOperator();
+    }
+
+    return builder.Conv2d(input, convWeights, fusedOptions.AsPtr());
+}
+
+const wnn::Operand SuperResolution::LoadNchw(const wnn::GraphBuilder& builder, bool softmax) {
+    const wnn::Operand input = utils::BuildInput(builder, "input", {1, 1, 224, 224});
+
+    // TODO: Does kernel_shape (5,5) need to be specified?
+    utils::Conv2dOptions conv1Options;
+    conv1Options.strides = {1, 1};
+    conv1Options.padding = {2, 2, 2, 2};
+    conv1Options.dilations = {1, 1};
+    const wnn::Operand conv1 =
+        BuildConv(builder, input, /*convIndex*/ 1, /*relu*/ true, &conv1Options);
+
+    // TODO: Does kernel_shape (3,3) need to be specified?
+    utils::Conv2dOptions conv2Options;
+    conv2Options.strides = {1, 1};
+    conv2Options.padding = {1, 1, 1, 1};
+    conv2Options.dilations = {1, 1};
+    const wnn::Operand conv2 =
+        BuildConv(builder, conv1, /*convIndex*/ 2, /*relu*/ true, &conv2Options);
+
+    // TODO: Does kernel_shape (3,3) need to be specified?
+    utils::Conv2dOptions conv3Options;
+    conv3Options.strides = {1, 1};
+    conv3Options.padding = {1, 1, 1, 1};
+    conv3Options.dilations = {1, 1};
+    const wnn::Operand conv3 =
+        BuildConv(builder, conv2, /*convIndex*/ 3, /*relu*/ true, &conv3Options);
+
+    // TODO: Does kernel_shape (3,3) need to be specified?
+    utils::Conv2dOptions conv4Options;
+    conv4Options.strides = {1, 1};
+    conv4Options.padding = {1, 1, 1, 1};
+    conv4Options.dilations = {1, 1};
+    const wnn::Operand conv4 =
+        BuildConv(builder, conv3, /*convIndex*/ 4, /*relu*/ false, &conv4Options);
+
+    const std::vector<int32_t> newShape1 = {-1, 1, 3, 3, 224, 224};
+    const wnn::Operand reshape1 = builder.Reshape(conv4, newShape1.data(), newShape1.size());
+
+    wnn::TransposeOptions transpose1Options;
+    std::vector<int32_t> permutation = {0, 1, 4, 2, 5, 3};
+    transpose1Options.permutation = permutation.data();
+    transpose1Options.permutationCount = permutation.size();
+    const wnn::Operand transpose1 = builder.Transpose(reshape1, &transpose1Options);
+
+    const std::vector<int32_t> newShape2 = {-1, 1, 672, 672};
+    const wnn::Operand reshape2 = builder.Reshape(transpose1, newShape2.data(), newShape2.size());
+
+    const wnn::Operand output = reshape2;
+    return output;
```

Comment:
Merge L.118 and L.121 to `return builder.Reshape(transpose1, newShape2.data(), newShape2.size()); ` that is output operand.

Reply:
Done.